test/Microsoft.Framework.Configuration.Json.Test/JsonConfigurationSourceTest.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Microsoft.Framework.Configuration
 {
-    public class JsonConfigurationSourceTest
+    public class JsonConfigurationProviderTest
```

Comment:
Please rename in one pattern to maintain consistency among all ConfigurationProviderTest files.